pandas/tests/groupby/test_function.py

```diff
@@ -1234,3 +1234,26 @@ def test_groupby_sum_timedelta_with_nat():
     res = gb["b"].sum(min_count=2)
     expected = Series([td3, pd.NaT], dtype="m8[ns]", name="b", index=expected.index)
     tm.assert_series_equal(res, expected)
+
+
+def test_groupby_empty_dataset():
+    # GH#41575
+    df = DataFrame(columns=["A", "B", "C"], dtype=int)
+
+    result = (
+        df.iloc[:0]
+        .groupby("A")
+        .B.describe(percentiles=[0.10, 0.20, 0.30], include="all")
```

Comment:
Can you parametrize this test:

  @pytest.mark.parametrize("kwargs", [{"percentiles": [0.10, 0.20, 0.30], "include": "all"}

Then each call to `describe(...)` becomes `describe(**kwargs)`. Also add a few different combinations of percentiles, include, and exclude.

Reply:
Sure👍 . Have added